### DISCUSSION TITLE
How to change notification transparency/opacity

### DISCUSSION THREAD

**Discussion by DivineMK** [prasanthrangan/hyprdots#1946] [Q&A] [answered]
I need help modifying this as it makes text hard to read. It seems like this is in the dunst.conf but I don't know where I should change it when using HyDE wallbash tools for theming.

**Comment by kRHYME7** [comment]
It's a layer so you can modify that layer in `userpref.conf`


something 

```
layerrule = unset, dunst
# Add stuff here 

```

check the Hyprland wiki to do it



https://wiki.hyprland.org/Configuring/Window-Rules/#layer-rules

**Reply by DivineMK** [reply]
Thank you for your quick response! However, it doesn't seem to have effect on dunst notifications. I tried to unset layerrule for notifications instead as well, which is what shown using hyprctl layers, but still doesn't work. Also I want to ask how to know what are layers, not windows? Should I try out with hyprctl clients/layers or is there a better way?

**Reply by kRHYME7** [reply]
anything that does not act as a window are layers on Wayland. And It's a Wayland thing.
e.g. Bars, notification, launchers

Imagine in 3d space You have
Background,Bottom,Top,Overlay

Wallpapers will be in the background.

Windows sets in the middle of the Bottom and Top layer.

bars Mostly go to Bottom or top

Notifications most likely be on overlay

Launchers can be on top or Overlay 

and so on

**Reply by kRHYME7** [reply]
Try this and remove a rule one by one to figure out which rules works.  I guess blur can help add opacity

``` hyprlang
layerrule = unset,dunst
layerrule = blur,dunst
layerrule = blurpopups,dunst
layerrule = ignorezero,dunst
```

**Reply by DivineMK** [reply]
This doesn't work. I confirm that because dunst is using RGBA, and removing the A (alpha) part from the colors work. How do I make this work with the HyDE system though, I currently change directly from dunstrc.

**Reply by kRHYME7** [reply]
change it in dunst.conf then Run `Hyde reload` to generate the autogenerated  stuff eg  dunstrc.

**Reply by DivineMK** [reply, marked as answer]
Thank you for the suggestion. However, this also doesn't seem to work as the wallbash.conf settings override settings from dunst.conf. I figured the way from your suggestions tho. I found out that I need to change ~/.config/hyde/wallbash/Wall-Ways/notification/dunst.dcol so that the change persists.
